static_src/actions/error_actions.js

```diff
@@ -0,0 +1,9 @@
+
+import errorActionTypes from '../constants.js';
+import Dispatcher from '../dispatcher.js';
+
+export default {
+  errorFetch() {
```

Comment:
Yea pretty much. It would be "global" error handling, so if we want a generic message when requests are failing, we would be using the error actions.